packages/eslint-plugin/src/configs/eslint-recommended.ts

```diff
@@ -0,0 +1,49 @@
+/**
+ * The goal of this ruleset is to update the eslint:recommended config to better
+ * suit Typescript. There are two main reasons to change the configuration:
+ * 1. The Typescript compiler natively checks some things that therefore don't
+ *    need extra rules anymore.
+ * 2. Typescript allows for more modern Javascript code that can thus be
+ *    enabled.
+ */
+export default {
+  overrides: [
+    {
+      files: ['*.ts', '*.tsx'],
+      rules: {
+        /**
+         * 1. Disable things that are checked by Typescript
+         */
+        //Checked by Typescript - ts(2378)
+        'getter-return': false,
```

Comment:
I'm not sure if this works or not because `false` is invalid value. Any of `0`, `1`, `2`, `"off"`, `"warn"`, and `"error"` are allowed.

Reply:
I have fixed this in master: https://github.com/typescript-eslint/typescript-eslint/commit/d52a6839d3e9af3952882d9b4ac9de5b12fdaf16